templates/admin/conference/ticket/change_form.html

```diff
@@ -0,0 +1,14 @@
+{% extends "admin/change_form.html" %}
+{% block object-tools %}
+    <ul class="object-tools">
+        <li>
+            <a class="historylink" href="history/">History</a>
+        </li>
+        <li>
+            <a class="" href="{% url 'admin:assopy-invoice-associated-order' invoice_id=original.orderitem.invoice.id %}">Order</a>
```

Comment:
The `assopy-invoice-associated-view` takes the invoice id, finds the associated order to and order id and redirects the user to `admin:assopy_order_change`  - you can instead access the `Order` instance directly from the orderitem and have the anchor point to `admin:assopy_order_change` directly. This will also fix the issue with the page not rendering when there is no invoice associated with an order.

Reply:
Thanks for the advice! That fixed it!